.github/workflows/build-opencv-base-container.yml

```diff
@@ -18,56 +18,56 @@ on:
     - build/containers/intermediate/Dockerfile.opencvsharp-build
     - build/intermediate-containers.mk
     - Makefile
-  pull_request_review_comment:
+  issue_comment:
     types: [created]
+    branches: [ main ]
 
 env:
   AKRI_COMPONENT: opencvsharp-build
   MAKEFILE_COMPONENT: opencv-base
 
 jobs:
 
-  add-label-missing-comment:
+  add-build-label-to-pr:
     runs-on: ubuntu-latest
-    if: contains(github.event.pull_request.labels.*.name, 'build-dependency-containers') == false
+    if: github.event_name == 'issue_comment' && contains(github.event.comment.body, '/add-build-dependency-containers')
     steps:
     - uses: actions/checkout@v2
-    - name: Add comment about missing build-dependency-containers label
-      uses: actions/github-script@v4
+    - name: Add build-dependency-containers label
+      uses: actions/github-script@0.9.0
       if: success()
       with:
         github-token: ${{secrets.GITHUB_TOKEN}}
         script: |
+          github.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ['build-dependency-containers']
+          })
           github.issues.createComment({
-            issue_number: context.payload.pull_request.number,
+            issue_number: context.issue.number,
             owner: context.repo.owner,
             repo: context.repo.repo,
-            body: 'hey this apparently should run but running it will take hours. Do you want to run it? If so, add flag build deps cont. (also tags Akri Maintainers)'
+            body: '👋 Added build-dependency-containers label :)!'
           })
-          exit 1
 
-  add-build-label-to-pr:
+  add-label-missing-comment:
     runs-on: ubuntu-latest
-    if: contains(github.event.comment.body, '/add-build-dependency-containers') && github.event_name == 'pull_request_review_comment'
+    if: github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'build-dependency-containers') == false

```

Comment:
will this add the missing comment every time the PR is udpated?

Reply:
This will add the comment every time the PR is updated until the "build-dependency-containers" flag is set.

Perhaps we should add the comment only the first time the PR is created without the label?